src/open_samus_returns_rando/specific_patches/door_patches.py

```diff
@@ -39,7 +39,7 @@
     "unk2": 1,
     "unk3": 0,
     "args": Container({
-        "601445949": Container({
```

Comment:
Oopsie.
I was curious what is the string for this CRC32 and it's "CallbackName". Could use the string instead of the cryptic value.